pkg/collector/corechecks/snmp/devicecheck/devicecheck.go

```diff
@@ -22,9 +22,9 @@ import (
 )
 
 const (
-	snmpLoaderTag    = "loader:core"
-	serviceCheckName = "snmp.can_check"
-
+	snmpLoaderTag        = "loader:core"
+	serviceCheckName     = "snmp.can_check"
+	deviceHostnamePrefix = "device:"
```

Comment:
I don't belive colon is allowed in hostnames?

Reply:
Yes! Or, it worked with some tests we ran. Some info are available on JIRA II-505